<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
	<class>GraphViewClassClass</class>
	<widget class="QMainWindow" name="GraphViewClassClass">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>1536</width>
				<height>863</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>QtApplication1</string>
		</property>
		<property name="styleSheet">
			<string notr="true">
				background-color: rgb(243, 243, 243);
				background-color: rgb(255, 255, 255);
				background-color:rgb(34, 34, 34)


			</string>
		</property>
		<widget class="QWidget" name="centralWidget">
			<widget class="QWidget" name="widget" native="true">
				<property name="geometry">
					<rect>
						<x>0</x>
						<y>0</y>
						<width>1536</width>
						<height>80</height>
					</rect>
				</property>
				<property name="styleSheet">
					<string notr="true">
						background-color: rgb(53,50,84);
						background-color: rgba(75, 0, 0, 200);
						background-color: rgb(34,34,34);
					</string>
				</property>
				<widget class="QLabel" name="label_10">
					<property name="geometry">
						<rect>
							<x>70</x>
							<y>20</y>
							<width>91</width>
							<height>41</height>
						</rect>
					</property>
					<property name="styleSheet">
						<string notr="true">
							font-weight: 400;
							font: 20pt &quot;Bahnschrift Condensed&quot;;
						</string>
					</property>
					<property name="text">
						<string>Road-Map</string>
					</property>
				</widget>
				<widget class="QLabel" name="label_4">
					<property name="geometry">
						<rect>
							<x>20</x>
							<y>20</y>
							<width>31</width>
							<height>31</height>
						</rect>
					</property>
					<property name="text">
						<string/>
					</property>
					<property name="pixmap">
						<pixmap resource="QtApplication1.qrc">:/newPrefix/img/pin.png</pixmap>
					</property>
					<property name="scaledContents">
						<bool>true</bool>
					</property>
				</widget>
			</widget>
			<widget class="Line" name="line">
				<property name="geometry">
					<rect>
						<x>0</x>
						<y>80</y>
						<width>1536</width>
						<height>3</height>
					</rect>
				</property>
				<property name="styleSheet">
					<string notr="true">background-color: rgb(75, 75, 75);</string>
				</property>
				<property name="orientation">
					<enum>Qt::Orientation::Horizontal</enum>
				</property>
			</widget>
			<widget class="QGraphicsView" name="graphicsView">
				<property name="geometry">
					<rect>
						<x>300</x>
						<y>110</y>
						<width>1221</width>
						<height>461</height>
					</rect>
				</property>
				<property name="styleSheet">
					<string notr="true">background-color: rgb(255, 255, 255);</string>
				</property>
			</widget>
			<widget class="QPushButton" name="addCityButton">
				<property name="geometry">
					<rect>
						<x>10</x>
						<y>170</y>
						<width>131</width>
						<height>41</height>
					</rect>
				</property>
				<property name="text">
					<string>Add City</string>
				</property>
			</widget>
			<widget class="QPushButton" name="deletecitybutton">
				<property name="geometry">
					<rect>
						<x>150</x>
						<y>170</y>
						<width>131</width>
						<height>41</height>
					</rect>
				</property>
				<property name="text">
					<string>Delete City</string>
				</property>
			</widget>
			<widget class="QLineEdit" name="cityNameLineEdit">
				<property name="geometry">
					<rect>
						<x>12</x>
						<y>113</y>
						<width>271</width>
						<height>41</height>
					</rect>
				</property>
				<property name="focusPolicy">
					<enum>Qt::FocusPolicy::WheelFocus</enum>
				</property>
				<property name="alignment">
					<set>Qt::AlignmentFlag::AlignCenter</set>
				</property>
				<property name="placeholderText">
					<string>Enter City Name</string>
				</property>
			</widget>
			<widget class="QPushButton" name="addEdgeButton">
				<property name="geometry">
					<rect>
						<x>10</x>
						<y>220</y>
						<width>131</width>
						<height>41</height>
					</rect>
				</property>
				<property name="text">
					<string>Add Edge</string>
				</property>
			</widget>
			<widget class="QPushButton" name="deleteedgebutton">
				<property name="geometry">
					<rect>
						<x>150</x>
						<y>220</y>
						<width>131</width>
						<height>41</height>
					</rect>
				</property>
				<property name="text">
					<string>Delete Edge</string>
				</property>
			</widget>
			<widget class="QPushButton" name="pushButton_5">
				<property name="geometry">
					<rect>
						<x>10</x>
						<y>270</y>
						<width>271</width>
						<height>41</height>
					</rect>
				</property>
				<property name="text">
					<string>Delete All Graph</string>
				</property>
			</widget>
			<widget class="QPushButton" name="bfsbutton">
				<property name="geometry">
					<rect>
						<x>10</x>
						<y>370</y>
						<width>271</width>
						<height>41</height>
					</rect>
				</property>
				<property name="text">
					<string>Display BFS</string>
				</property>
			</widget>
			<widget class="QPushButton" name="dfsbutton">
				<property name="geometry">
					<rect>
						<x>10</x>
						<y>420</y>
						<width>271</width>
						<height>41</height>
					</rect>
				</property>
				<property name="text">
					<string>Display DFS</string>
				</property>
			</widget>
			<widget class="QPushButton" name="mspbutton">
				<property name="geometry">
					<rect>
						<x>10</x>
						<y>520</y>
						<width>271</width>
						<height>41</height>
					</rect>
				</property>
				<property name="text">
					<string>Display Minimum Spanning Tree</string>
				</property>
			</widget>
			<widget class="QLineEdit" name="startCitybutton">
				<property name="geometry">
					<rect>
						<x>10</x>
						<y>320</y>
						<width>271</width>
						<height>41</height>
					</rect>
				</property>
				<property name="focusPolicy">
					<enum>Qt::FocusPolicy::ClickFocus</enum>
				</property>
				<property name="alignment">
					<set>Qt::AlignmentFlag::AlignCenter</set>
				</property>
				<property name="placeholderText">
					<string>Enter Start City Name</string>
				</property>
			</widget>
			<widget class="QPushButton" name="pushButton_9">
				<property name="geometry">
					<rect>
						<x>10</x>
						<y>470</y>
						<width>271</width>
						<height>41</height>
					</rect>
				</property>
				<property name="text">
					<string>Display Dijkistra</string>
				</property>
			</widget>
			<widget class="QPushButton" name="undobutton">
				<property name="geometry">
					<rect>
						<x>10</x>
						<y>570</y>
						<width>131</width>
						<height>41</height>
					</rect>
				</property>
				<property name="text">
					<string>Undo</string>
				</property>
			</widget>
			<widget class="QPushButton" name="redobutton">
				<property name="geometry">
					<rect>
						<x>150</x>
						<y>570</y>
						<width>131</width>
						<height>41</height>
					</rect>
				</property>
				<property name="text">
					<string>Redo</string>
				</property>
			</widget>
			<widget class="QWidget" name="path" native="true">
				<property name="geometry">
					<rect>
						<x>300</x>
						<y>600</y>
						<width>1221</width>
						<height>181</height>
					</rect>
				</property>
			</widget>
			<widget class="QPushButton" name="returntograph">
				<property name="geometry">
					<rect>
						<x>10</x>
						<y>620</y>
						<width>271</width>
						<height>51</height>
					</rect>
				</property>
				<property name="text">
					<string>Return to Graph</string>
				</property>
			</widget>
		</widget>
	</widget>
	<layoutdefault spacing="6" margin="0"/>
	<resources>
		<include location="QtApplication1.qrc"/>
		<include location="../../../../.designer/backup/QtApplication1.qrc"/>
	</resources>
	<connections/>
</ui>
